test/e2e/operator_test.go

```
@@ -106,12 +106,13 @@ func TestRedeployOnConfigChange(t *testing.T) {
 }
 
 func TestEncryptionTypeAESCBC(t *testing.T) {
+	ctx := context.TODO()
 	// make sure that deployment is not in progress before and after the test
 	cs := library.GetClients(t)
 	libraryapi.WaitForAPIServerToStabilizeOnTheSameRevision(t, cs.Kube.CoreV1().Pods(operatorclient.TargetNamespace))
```

golangci-lint (2.12.2)

[error] Error return value of `libraryapi.WaitForAPIServerToStabilizeOnTheSameRevision` is not checked

(errcheck)